decidim-forms/app/views/decidim/forms/admin/questionnaires/edit.html.erb

```diff
@@ -4,11 +4,7 @@
   <%= render partial: "decidim/templates/admin/questionnaire_templates/choose", locals: { target: questionnaire, form_title: t("decidim.forms.admin.questionnaires.edit.title") } %>
 <% else %>
   <%= decidim_form_for(@form, url: update_url, method: :put, html: { class: "form edit_questionnaire" }) do |form| %>
-    <%= render partial: "decidim/forms/admin/questionnaires/form",
-               object: form,
-               locals: {
-                 title: templates_defined ? t("form.title", scope: "decidim.templates.admin.questionnaire_templates") : t(".title")
-    } %>
+    <%= render partial: "decidim/forms/admin/questionnaires/form", object: form, locals: { title: title_for_questionnaire(templates_defined) } %>
```

Comment:
```suggestion
    <%= render partial: "decidim/forms/admin/questionnaires/form", object: form, locals: { title: title_for_questionnaire } %>
```
Maybe we could just add all the `defined?` stuff into the helper and keep the view simpler?